tests/acceptance/test_dcos_package.py

```diff
@@ -24,8 +24,8 @@ def test_uninstall_package_and_wait():
     uninstall_package_and_wait('chronos')
     delete_zk_node('chronos')
     assert package_installed('chronos') == False
-    # this should only pass if we can install after the wait
-    install_package_and_wait('chronos')
+#    # this should only pass if we can install after the wait
+#    install_package_and_wait('chronos')
```

Comment:
this looks like a CLI bug:  https://github.com/dcos/dcos-cli/blob/master/dcos/packagemanager.py#L34-L35
I will be filing a JIRA bug.